config/database.yml.gitpod

```diff
@@ -1,10 +1,6 @@
 development:
-  adapter: mysql2
-  username: root
-  password: 
-  encoding: utf8
-  database: plots
-  strict: false
+  adapter: sqlite3
```

Comment:
Changing database might be dangerous :thinking:  I'll  wait for the other guys review

Reply:
@alaxalves since this is only specific to gitpod and we just started using it...what issues do you anticipate we will encounter? I know we have folks who use sqlite3 for development and the only drawback is that some tests fail locally. But you are more equipped  in the devops stuff and I would love to hear your thoughts thanks :smiley:

Reply:
Basically here, we started with an sqlite3 db, but our app is complex enough that i got locked database errors pretty often upon initial startup. So I thought perhaps we ought to try to reproduce our entire production environment in the dev environment. This may also be important because we use some MySQL features such as full-text search!